readme.md

```diff
@@ -0,0 +1,35 @@
+# Crypto-Transction Backend
+
+### About
+
+- This is the backend of crypto-transactions application.
+
+### Tech
+
+It uses a number of open source projects to work properly:
+
+- NodeJS
+- ExpressJS
+- MongoDB
+
+And of course it is itself open source with a public repository
+on GitHub.
+
+## Installation
+
+- Make sure you have NodeJS 16.0+ installed lcoally on your device
+- Create a .env file in the root directory of the project and copy the contents of .env.example into it
+- Create a .env file in the root directory of the project and copy the contents of .env.example into it.
+- Install dependencies using the following command
+
+```
+npm i
+```
+
+- Start the application using the following command
+
+```
+npm start
+```
+
```

Comment:
Can you please write the instructions to run in development and production mode.